src/Utilities/functions.jl

```diff
@@ -121,6 +120,15 @@ function Base.getindex(it::ScalarFunctionIterator{MOI.VectorAffineFunction{T}},
     MOI.VectorAffineFunction(terms, constant)
 end
 
+termindices(t::MOI.ScalarAffineTerm) = (t.variable_index.value,)
```

Comment:
There should be a comment indicating what it returns

Reply:
Sure